tests/kvmtest.sh

```diff
@@ -145,7 +145,8 @@ test_features.py \
 test_procdockerstatsd.py \
 iface_namingmode/test_iface_namingmode.py \
 platform_tests/test_cpu_memory_usage.py \
-bgp/test_bgpmon.py"
+bgp/test_bgpmon.py" \
+container_checker/test_container_checker.py
```

Comment:
The above two lines should be:
```
bgp/test_bgpmon.py \
container_checker/test_container_checker.py"
```
Otherwise, the kvmtest.sh script will try to run `container_checker/test_container_checker.py` and will fail.

Reply:
Good catch! Fixed.